### PR TITLE
refactor: move collector contents logic to CollectorContents from CollectorMainPage

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

There was a bug that the filters from query string were not applied to api call. 

To solve this problem, it was inevitable to refactor because it was pretty hard to figure out when and where the filters are initialized.

This PR separate concerns:
- CollectorMainPage component cares only three things. -
  - converting values between page url query string and store states
  - collector list empty case
  - props drilling if components requires
- CollectorContents component cares the things that contents area should handle. - 
  - props drilling if children components requires and event handler binding

### CODE REVIEW

**Review by skdud4659 — APPROVED** [review]
LGTM!! Thanks for your hard work!

File: apps/web/src/services/asset-inventory/collector/collector-main/CollectorMainPage.vue [hunk 1]
```diff
@@ -45,143 +42,116 @@
...
+    provider: collectorPageState.selectedProvider,
+    filters: collectorPageState.searchFilters,
+}));
+
+watchDebounced(collectorMainPageQueryValue, async (queryValue) => {
+    const newQuery: CollectorMainPageQuery = {
+        provider: primitiveToQueryString(queryValue.provider),
+        filters: urlFilterConverter.setFilters(queryValue.filters ?? []).rawQueryStrings,
+    };
+    console.debug('newQuery', queryValue.provider, newQuery);
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-main/CollectorMainPage.vue:115]: quibble: here!

**Review by yuda110 — APPROVED** [review]
It seems like `CollectorContents.vue` has gone through several refactoring already. This PR looks good.

File: apps/web/src/services/asset-inventory/collector/collector-main/CollectorMainPage.vue (hunk 1 above)
Comment (WANZARGEN) [reply apps/web/src/services/asset-inventory/collector/collector-main/CollectorMainPage.vue]: Thank u!

**Pull request merged**